docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -125,6 +132,56 @@ show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
 
 This property is optional; the default is ``true``.
 
+``kafka.security-protocol``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable ``SSL`` for the Kafka connection, set this property to ``SSL``.
+
+This property is optional; the default ist ``PLAINTEXT``.
+
+``kafka.keystore-location``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Location of the keystore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.keystore-password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Password for the keystore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.key-password``
```

Comment:
We are missing keystore and truststore types, aren't we?

Reply:
Sorry I didn't rebase my commits back then. I wasn't aware of co-author GH feature. I just merged both commits.